pysmt/walkers/identitydag.py

```diff
@@ -73,11 +73,13 @@ def walk_lt(self, formula, args, **kwargs):
         return self.mgr.LT(args[0], args[1])
 
     def walk_forall(self, formula, args, **kwargs):
-        qvars = [self.walk_symbol(v) for v in formula.quantifier_vars()]
```

Comment:
Why this change (and the ones below)? Doesn't this prevent proper sub-classing of IdentityDagWalker?
For example, if I define my own walk_symbol (that for example rewrites all the integer variables as real variables) this code would create the wrong formula unless I explicitly override `walk_forall`...